From 4c7739e4645d022edd7e0e9777e5f6e3d13160ac Mon Sep 17 00:00:00 2001
From: Song Bing <bing.song@nxp.com>
Date: Thu, 30 Aug 2018 14:26:11 -0700
Subject: [PATCH 15/75] v4l2videodec: Remove sink event task stop as flush()
 will do the same.

https://bugzilla.gnome.org/show_bug.cgi?id=752962

Upstream-Status: Pending

Signed-off-by: Song Bing<bing.song@nxp.com>
(cherry picked from commit 2ebc731916be1fb427a0afb774f517dc67379cf5)
Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 sys/v4l2/gstv4l2videodec.c | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/sys/v4l2/gstv4l2videodec.c b/sys/v4l2/gstv4l2videodec.c
index 1a74010..18cbb5d 100644
--- a/sys/v4l2/gstv4l2videodec.c
+++ b/sys/v4l2/gstv4l2videodec.c
@@ -1198,28 +1198,8 @@ gst_v4l2_video_dec_sink_event (GstVideoDecoder * decoder, GstEvent * event)
   gboolean ret;
   GstEventType type = GST_EVENT_TYPE (event);
 
-  switch (type) {
-    case GST_EVENT_FLUSH_START:
-      GST_DEBUG_OBJECT (self, "flush start");
-      gst_v4l2_object_unlock (self->v4l2output);
-      gst_v4l2_object_unlock (self->v4l2capture);
-      break;
-    default:
-      break;
-  }
-
   ret = GST_VIDEO_DECODER_CLASS (parent_class)->sink_event (decoder, event);
 
-  switch (type) {
-    case GST_EVENT_FLUSH_START:
-      /* The processing thread should stop now, wait for it */
-      gst_pad_stop_task (decoder->srcpad);
-      GST_DEBUG_OBJECT (self, "flush start done");
-      break;
-    default:
-      break;
-  }
-
   return ret;
 }
 
-- 
2.43.0

